upgrade_common_services.sh

```diff
@@ -148,6 +150,101 @@ function check_preqreqs() {
     fi
 }
 
+function pre_zen(){
+    local csNS=$1
+
+    STEP=$((STEP + 1 ))
+    msg ""
+    title "[${STEP}] Detecting Operator Condition of zen operator v1.4.2 in ${csNS} namespace..."
+    msg "-----------------------------------------------------------------------"
+
+    if oc get operatorcondition -n ${csNS} 2>/dev/null | grep ibm-zen-operator ; then
+        msg "Removing Operator Condition of zen operator v1.4.2..."
+        list=$(oc get operatorcondition -o custom-columns=operatorcondition:.metadata.name --no-headers -n ${csNS} | grep ibm-zen-operator 2>/dev/null)
+        for name in ${list};do
+            if [ ! -z "${name}" ] && [[ "${name}" =~ ibm-zen-operator.v1.4.2 ]]; then
+                oc delete operatorcondition ${name} -n ${csNS} || true
+            fi
+        done
+    else
+        msg "Operator Condition in namespace ${csNS} not found, skipping..."
+    fi
+}
+
+function zenopr_check() {
+    local csNS=$1
+
+    STEP=$((STEP + 1 ))
+    msg ""
+    title "[${STEP}] Checking if Zen Operator has been upgraded to the middle version..."
```

Comment:
Could we update the msg to `Waiting for IBM Zen Operator upgrading to latest version in the current channel ${currentChannel}`